zero_shot_classification/README.MD

```diff
@@ -0,0 +1,24 @@
+## Zero-Shot Classification service   
+Zero-Shot using Transformers on AWS Lambda. Check root readme for complete setup info.
+
+## Request format
+```
+{
+        "texts": ["food was great", "food was bad", "i am going out for food"],
+        "labels": ["negative", "postive", "neutral"],
+        "hypothesis": "The sentiment of the review is {}.",
+        "model_name": "typeform/mobilebert-uncased-mnli",
+        "tokenizer_name": "typeform/mobilebert-uncased-mnli"
```

Comment:
If these 4 are not passed, it should use the default values.
Also add `multi_label` in config and set it to False. Request can also have non-mandatory `multi_label`